source/manual/add-a-new-document-type.html.md

```diff
@@ -6,60 +6,49 @@ layout: manual_layout
+The [document type][] describes what a page on GOV.UK looks like.
 
-## Add the document type in the govuk-content-schema repo
+[document type]: https://docs.publishing.service.gov.uk/document-types.html
 
-You need to add the document type into the
-[allowed document types][allowed-document-types] in
-[govuk-content-schemas][govuk-content-schemas].
+## Add the document type in the govuk-content-schema repo
 
-Once you have added the document type you should:
+You need to add the document type into the [allowed document types][] in [govuk-content-schemas][]. Once you have added the document type you should:
 
-- commit the change
-- run `bundle exec rake` to generate the schemas again
-- commit this update separately
+- Commit the change
+- Run `bundle exec rake` to generate the schemas again
+- Commit this update separately
 
 Examples of implementation:
 
-- [https://github.com/alphagov/govuk-content-schemas/pull/652][pr-652]
-- [https://github.com/alphagov/govuk-content-schemas/pull/630][pr-630]
+- <https://github.com/alphagov/govuk-content-schemas/pull/652>
+- <https://github.com/alphagov/govuk-content-schemas/pull/630>
 
-## Create a new rake task in the publishing app
+[allowed document types]: https://github.com/alphagov/govuk-content-schemas/blob/main/lib/govuk_content_schemas/allowed_document_types.yml
+[govuk-content-schemas]: https://github.com/alphagov/govuk-content-schemas
 
-The rake task will publish the route using the new document type.
+## Add a new content schema
 
-Before you begin, you must identify which publishing and rendering applications
-will use the document type. You need to include these as parameters (payload)
-when you generate the route.
+If your document type needs a new content schema, see "[Adding a new schema][]".
 
-Examples of implementation:
+[Adding a new schema]: https://github.com/alphagov/govuk-content-schemas/blob/main/docs/adding-a-new-schema.md
 
-- [https://github.com/alphagov/collections-publisher/pull/250][pr-250]
+## Make the new document type available to search
 
-You should publish your new content item with your document type to the
-Publishing API, especially if you want it to be part of search, as it makes it
-easy to integrate search this way.
+If your document type should be available in search results, see "[Make a new document type available to search][]".
 
-> **Note**
->
-> Having a document type with `placeholder` prefix will not publish routes. More
-> information can be found here under [placeholder items][placeholder-items].
+[Make a new document type available to search]: /manual/make-a-new-document-type-available-to-search.html
 
-## Add a new content schema
+## Create a new Rake task in the publishing app
 
-See "[Adding a new schema][adding-a-new-schema]"
+If your new document type will be published as a static route (rather than from a publishing app), you'll need a Rake task will publish the route using the new document type.
```

Comment:
```suggestion
If your new document type will be published as a static route (rather than from a publishing app), you'll need to create a Rake task to publish the route using the new document type.
```